app/api/debug/posts/route.ts

```
@@ -0,0 +1,175 @@
+        if (authorMatch && authorMatch[1]) {
+          const authorId = authorMatch[1];
+          if (!authors[authorId]) {
+            authorIssues[filename] = `References non-existent author: ${authorId}`;
+          }
+        } else {
+          authorIssues[filename] = 'No author found in frontmatter';
+        }
+      } catch (error) {
+        authorIssues[filename] = `Error checking author: ${error instanceof Error ? error.message : String(error)}`;
+      }
+    });
```

Biome (1.9.4)

[error] Prefer for...of instead of forEach.

forEach may lead to performance issues when working with large arrays. When combined with functions like filter or map, this causes multiple iterations over the same type.

(lint/complexity/noForEach)